ui_sim.py

```diff
@@ -107,6 +111,17 @@ def __init__(self, map_width, map_height, sim, omsgr, master=None, logger=None):
         )
         self.btnDisplayPoints.pack(fill=tk.BOTH, expand=True, side=tk.LEFT)
 
+        self.topRightFrame = tk.Frame(master=self.logFrame)
+        self.topRightFrame.pack(fill=tk.NONE, expand=False, side=tk.TOP, anchor="ne")
+
+        self.btnDisplayScoreboard = ttk.Button(
+            master=self.topRightFrame,
+            text="Finish Game",
```

Comment:
do we need this button if scoreboard is shown for a win case? i think it might be good to leave (in case a win state is not possible to reach based on the scripts) but i want to check that others have different thoughts on it